### DISCUSSION TITLE
rinkeby faucet offline

### DISCUSSION THREAD

**Discussion by tanuj-rai** [smartcontractkit/full-blockchain-solidity-course-py#492] [Q&A] [answered]
I have done every step for Ethereum in the Rinkeby test network account but I do not receive any ETH yet. Please help me to get ETH.

**Comment by priyansuapk** [comment]
I am having the same problem

**Comment by cromewar** [comment, marked as answer]
Yo can use the chainlink faucet because the official is mot working. Optionally If you paste your wallets here I'll send some Eth to rinkeby.

**Reply by programonauta** [reply]
> Yo can use the chainlink faucet because the official is mot working. Optionally If you paste your wallets here I'll send some Eth to rinkeby.

Hi, could you send me some eth to Rinkeby? 0x7254935f4B770882CD8f8C0044D5adD9a44d30f6

**Reply by cromewar** [reply]
Sent ;) https://rinkeby.etherscan.io/tx/0xc319013a9fb39a673a892d29f373a4834b71ef386aa42b3c83c93b707de82d7a

**Comment by tanuj-rai** [comment]
Wallet Address- "0x8380b87c02A2ea4929718a595CC4Ea1320F1248f"
thank you very much for the help

**Reply by cromewar** [reply]
Sent: https://rinkeby.etherscan.io/tx/0xd4611c1d4479b112dd3b7d39ccda674032c95f7532a1b328f4cb43a907a4d3bc

**Reply by programonauta** [reply]
Thanks @cromewar

**Comment by tanuj-rai** [comment]
Got it, thank you!

**Comment by MaxMike23** [comment]
@cromewar I'm having trouble getting Rinkeby ETH too.
My wallet is https://rinkeby.etherscan.io/address/0x48A656424b835187C2c130c4BC0bA084368856ec

**Reply by cromewar** [reply]
sent: https://rinkeby.etherscan.io/tx/0x3550bc74bb43a38ee5a3bcf48a3ab7f0de1d2b28de5dce1a5c2d83cc967dd256

**Comment by tanuj-rai** [comment]
Actually, I wanted to close this thread because I am getting emails from
other persons also. so kindly please close this thread.

**Reply by cromewar** [reply]
it's closed now ;)
